### ISSUE TITLE
Bilinear Upsampling

### ISSUE THREAD

**Issue by Yan107351111** [IAmSuyogJadhav/3d-mri-brain-tumor-segmentation-using-autoencoder-regularization#50]
In the paper, the author specifies using bilinear interpolation to upsample the features in the decoder heads.
Going over your implementation, I saw you are using Kerase's UpSampling3D object to do that.
But it seems to me that what this object does is repeat elements, not perform interpolation.
Please, correct me if I'm wrong.

**Comment by FBehrad** [comment]
You are right. By default, the UpSampling2D layer in tensorflow will use a nearest neighbor algorithm to fill in the new rows and columns. This has the effect of simply doubling rows and columns.
